src/oc_erchef/apps/oc_chef_wm/src/chef_wm_status.erl

```diff
@@ -58,19 +59,33 @@ to_json(Req, State) ->
 
 -spec check_health() -> {pong | fail, binary()}.
 check_health() ->
+    Version = get_version(),
+
     Pings = spawn_health_checks(),
     Status = overall_status(Pings),
 
     log_failure(Status, Pings),
     KeyGen = chef_keygen_cache:status_for_json(),
     Indexing = chef_index:status(),
 
-    StatList = [{<<"status">>, ?A2B(Status)},
+    StatList = [{<<"version">>, Version},
+                {<<"status">>, ?A2B(Status)},
                 {<<"upstreams">>, {Pings}},
                 {<<"keygen">>, {KeyGen}},
                 {<<"indexing">>, {Indexing}}],
     {Status, chef_json:encode({StatList})}.
 
+get_version() ->
+    {ok, Device} = file:open(?VERSION_PATH, [read]),
+    %% Assuming that the first line of the file will have the version.
+    Version =
+        case file:read_line(Device) of
+            {ok, Data} -> list_to_binary(lists:subtract(Data,"chef-server \n"));
+            _ -> <<"error">>
+        end,
+    file:close(Device),
+    Version.
```

Comment:
I wonder if we might try to do this at statup so that we don't have to read the file on every request?

Reply:
got it. Will make the necessary changes